force-app/tests/Shared Code/TriggerHandler_Test.cls

```diff
@@ -103,15 +103,18 @@ private class TriggerHandler_Test {
     static void testNonTriggerContext() {
         try {
             handler.run();
-            System.assert(false, 'The handler ran but should have thrown');
+            System.Assert.isTrue(
```

Comment:
This is a good use case for `Assert.fail`

Reply:
Addressed!